### PR TITLE
DESY1wl tests

### LINKED ISSUE #92: Missing tests: DESY1wl
- [x] Read data from full catalog (instead of lite version)
- [x] Read shear maps from lite versions (i.e. generate them, write them and then read them again)
- [x] Test `get_nz`
- [x] Test `get_dtype`

### PR DESCRIPTION
**Pull request by carlosggarcia**
Closes #92

Missing tests of reading full catalog and save/read lite catalog

### CODE REVIEW

File: xcell/tests/test_mapper_desy1wl.py [hunk 1]
```diff
@@ -29,23 +37,73 @@ def get_es():
...
-    xc.mappers.MapperDESY1wl(config)
+    m_lite = xc.mappers.MapperDESY1wl(config)
+    cat_from_lite = m_lite.get_catalog()
+    s_from_lite = m_lite.get_signal_map()
+    psf_from_lite = m_lite.get_signal_map('PSF')
+    mask_from_lite = m_lite.get_mask()
+    assert np.all(cat == cat_from_lite)
+    assert np.all(np.array(s) == np.array(s_from_lite))
+    assert np.all(np.array(psf) == np.array(psf_from_lite))
+    assert np.all(mask == mask_from_lite)
```
Comment (damonge) [inline xcell/tests/test_mapper_desy1wl.py:103]: Can you add the coupled nl to this list? That way we check that the `w2s2` map is also read from lite.

**Review by damonge — CHANGES_REQUESTED** [review]
Just one small thing. Everything else looks great.

File: xcell/tests/test_mapper_desy1wl.py (hunk 1 above)
Comment (carlosggarcia) [reply xcell/tests/test_mapper_desy1wl.py:109]: Good catch. Done.

**Review by damonge — APPROVED** [review]
lgtm

**Pull request merged**